src/group/managed_group/test_managed_group.rs

```diff
@@ -0,0 +1,50 @@
+use crate::prelude::*;
+
+#[test]
+fn test_managed_group_persistence() {
```

Comment:
This is good for a first test. But we really need to extend our testing framework and make persistence always a part of it.

Reply:
Going forward we also should not use actual files for serialization, but rather in-memory buffers.

Reply:
either that our use temporary files (there's a tempfile crate for something like this)